Source Main 5.2/source/NewUIMixInventory.cpp

```diff
@@ -525,7 +525,7 @@ void CNewUIMixInventory::RenderFrame()
         g_pRenderText->SetTextColor(220, 220, 220, 255);
         g_pRenderText->SetBgColor(40, 40, 40, 128);
 
-        wchar_t szTempText[2][100];
+        wchar_t szTempText[2][100] = { 0 };
```

Comment:
This is the main fix for this PR, this resolves the text rendering issues with goblin (item mix) NPCs.  

Other diffs here are just to lessen the compile warnings related to some initialization.